_posts/2021-02-01-coalesce-dbt-summary.markdown

```diff
@@ -0,0 +1,288 @@
+
+### Coming of (the new) Age
+
+A common theme was the seismic shift in how teams collaborate with data at scale. There are so many factors at play, but some of the many called out include:
+
+* [The Rise of Cloud Data Warehouses](https://mechanicalrock.github.io/2020/11/09/the-cloud-data-warehouse.html) fuelling the shift from Extract, Transform, Load (ETL) towards ELT instead
+* Sustained growth and competition in the SAAS Data Extract and Load tooling market supporting the transition to Cloud Data Warehouses
+* The insatiable appetite within organisations for access to quality data to drive decision making
+* How DBT helps teams concurrently and safely deliver insights and quality data  
+
+Many presentations talked about the role of a data team, their relationship to the wider organisation, and why communication, trust and knowledge sharing are indispensable partners to technology;  a few included back-references to Justin Gage's popular post from 2019 titled [Data as a Product vs Data as a Service](https://medium.com/@itunpredictable/data-as-a-product-vs-data-as-a-service-d9f7e622dc55) and furnished further with their own experiences of pivoting from being a reactive service team to a pro-active partner.
+
+Notable in almost every presentation were the ability of teams to be more **self-reliant**, engage more diverse people and **skill-sets**, focus on the parts of their job they enjoyed more, and ultimately **serve their customers better**.
+
+
+### Collaboration and Knowledge Formation
+
+Never more so than in 2020 has the challenge of being co-located virtually, rather than physically, been more keenly felt.
+
+Many teams reflected on their strategies - beyond using collaborative workflows and tools promoting open-ness and transparency such as DBT - to enable continuity of learning and upskilling, in a year where many new recruits would on-board to companies remotely. In many cases, presenters affirmed their commitment to keeping documentation about their work, including the role that [DBT's self generating documentation](https://docs.getdbt.com/docs/building-a-dbt-project/documentation) played in communicating intent and understanding between teams and consumers. 
+
+In two very different talks, both Tristan Handy and Andrea Kopitz teach us that accepted knowledge is effectively the building of organisational consensus. Andrea's comment around creating some documentation - even if only temporarily useful - is often better than no documentation at all. It re-iterated to me also the importance of being assertive and descriptive in documentation to allow others to challenge, so that together we may reach a better consensus.
+
+On a more technical level, I took away from many talks the success teams had encountered by consolidating the knowledge and business logic of data domain and insight generation within their DBT projects, and how this had promoted more transparency and trust between consumers and producers / curators of data.
```

Comment:
Slightly cumbersome "I took away from many talks..."
How about:
"On a more technical level, many talks focussed on the success teams had..."

Reply:
Whole sentence could use some love- "between consumers and producers / curators of data" 
slash is clunky.